client/schemes/jellyfinScheme.ts

```diff
@@ -20,33 +21,46 @@ export default class JellyfinScheme {
     this.$auth = auth;
 
     this.options = Object.assign({}, {}, options);
+    this.mount();
   }
 
   _setToken(token: string): void {
     // Set Authorization token for all axios requests
     this.$auth.ctx.app.$axios.setHeader('X-Emby-Authorization', token);
   }
 
+  _setBaseUrl(serverUrl: string): void {
+    this.$auth.ctx.app.$axios.setBaseURL(serverUrl);
+  }
+
   _clearToken(): void {
     // Clear Authorization token for all axios requests
     this.$auth.ctx.app.$axios.setHeader('X-Emby-Authorization', false);
   }
 
   _setRememberMe(value: boolean): void {
     // Sets the remember me key which is used to relogin someone
-    this.$auth.$storage.setUniversal('rememberMe', value);
+    this.$auth.$storage.setState('rememberMe', value);
   }
 
   _getRememberMe(): boolean {
-    return this.$auth.$storage.getUniversal('rememberMe');
+    return destr(this.$auth.$storage.getState('rememberMe'));
   }
 
-  mounted(): Promise<never> {
-    const token = this.$auth.syncToken(this.name);
+  async mount(): Promise<void> {
```

Comment:
Having both `mount` and `mounted` is confusing.

What you're doing here is stuff that would fit in other, standard, Nuxt Auth methods.

Reply:
As said above, this is testing bits, will remove.